hiutil.c

```diff
@@ -36,12 +36,12 @@
 #include <stdlib.h>
 #include <string.h>
 #include <sys/types.h>
+#include <sys/time.h>
```

Comment:
```suggestion
#include <sys/time.h>
#include <sys/types.h>
```

Reply:
clang formatter fixes for failing CI job.

Reply:
Actually seems that the existing windows build job needed a check for `_MSC_VER`, updating above suggestion

Reply:
Yeah, noticed that in the CI too - seems an odd quirk.  The issue I encountered was a lack of a definition for 'struct timeval' ... I look more closely, see if this can be solved without conditional (else, +1 - let's go with what you have suggested there).

Reply:
> Do you think we should add a CI job for MinGW64 to make sure we don't break it again later?

Hmm, not sure - may be overkill as they're supposed to be essentially the same (in terms of headers), OTOH it would be nice to know in advance of any such issue.